mycli/main.py

```diff
@@ -475,7 +484,7 @@ def prompt_tokens(cli):
 
                 # Refresh the table names and column names if necessary.
                 if need_completion_refresh(document.text):
-                    self.refresh_dynamic_completions()
+                    self.refresh_completions(reset=need_completion_reset(document.text))
```

Comment:
Should you check if the command was successful before starting a thread to refresh completions? If the create/alter/update statement wasn't successful, wouldn't that be a waste of resources, especially if the database is large?